src/api/article/collection/controller.js

```diff
@@ -0,0 +1,46 @@
+import { checkIsFound } from 'utils/validation';
+import { sendJson } from 'utils/api';
+
+import ArticleCollection from './model';
+
+export const getAll = (req, res, next) =>
+  ArticleCollection.find({ active: true })
+    .then(checkIsFound)
+    .then(sendJson(res))
+    .catch(next);
+
+export const getOne = ({ params: { slug } }, res, next) =>
+  ArticleCollection.findOne({ slug, active: true })
+    .then(checkIsFound)
+    .then(sendJson(res))
+    .catch(next);
+
+export const create = async ({ body }, res, next) => {
```

Comment:
can you try to rewrite it with promises chain?

Reply:
Will try that later.
The rest of your comments are resolved.